packages/perseus/src/components/__stories__/multi-button-group.stories.tsx

```diff
@@ -2,65 +2,47 @@ import * as React from "react";
 
 import MultiButtonGroup from "../multi-button-group";
 
-type StoryArgs = {
-    allowEmpty: boolean;
-};
-
-type Story = {
-    title: string;
-    args: StoryArgs;
-};
+import type {PropsFor} from "@khanacademy/wonder-blocks-core";
+import type {Meta, StoryObj} from "@storybook/react";
 
-export default {
+const meta: Meta = {
     title: "Perseus/Components/Muli-Button Group",
+    component: MultiButtonGroup,
     args: {
         allowEmpty: true,
+        buttons: [],
+    },
+    render: function WithState(props: PropsFor<typeof MultiButtonGroup>) {
+        const [values, updateValues] = React.useState(props.values);
+        return (
+            <MultiButtonGroup
+                {...props}
+                values={values}
+                onChange={updateValues}
+            />
+        );
```

Comment:
Oh I like this! I usually put the `render` separately in each story and add their states individually, but I never thought about putting it directly in the meta so it applies to all of them.

Reply:
Right? I also saw this and it's really neat and makes ton of sense!

Reply:
I'm [lazy](http://wiki.c2.com/?LazinessImpatienceHubris) and always look for the least amount of work for myself! 😂